contracts/registry/DAOFeeRegistry.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+contract DAOFeeRegistry is Ownable {
+    address private feeRecipient;
+    uint256 private defaultFeeNumerator; // 1e4 = 100% fee
+    mapping(address => uint256) private rTokenFeeNumerator;
```

Comment:
let's create a `FEE_DENOMINATOR` constant?

Reply:
or return it in `getFeeDetails()`

Reply:
Yep, wanted to do that but I see that we use this 1e4 pattern elsewhere in the protocol so didn't do it assuming it was documented globally somewhere.

Reply:
Nothing specifically that this does assume that the same 1e4 denominator is used everywhere for revenue purposes.

Reply:
right -- forgot we already had a constant for it in `IDistributor`: `MAX_DISTRIBUTION`